### ISSUE TITLE
"Application output" inside IDE stops working once cleared

### ISSUE THREAD

**Issue by DavidKarlas** [mono/monodevelop#6316]
I can reproduce this on 7.8, Matt says he can reproduce also on 7.7.
Problem is once I use Clear button in Application output pad(see image below, middle button):
![image](https://user-images.githubusercontent.com/774791/46951208-9b54fa00-d087-11e8-9bc9-2995a0d62be2.png)
New output doesn't appear.

> VS bug [#705935](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/705935)

**Comment by mrward** [comment]
Possibly related to this change - https://github.com/mono/monodevelop/pull/5772/commits/92241d43d03221a075fe390de50bbec69aaea884

**Comment by mrward** [comment]
Repo:

1. Create a console project.
2. Add a while loop that writes output.
3. Project options - run configuration - disable use external console.
4. Run the project.
5. See count being written in Application Output.
6. Click the Clear button in the Application Output window.
7. No more output.

```
		public static void Main (string[] args)
		{
			int count = 0;
			while (true) {
				count++;
				Console.WriteLine (count);
				Thread.Sleep (500);
			}
		}
```

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-11] Comment by **xamarinc**



Fixed in version 7.8.0.1099 (master)



Pull Request #6533 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 479f1c73b696df54a75eaae2eabde35a55f1753c (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 7.7.0.1811 (release-7.7)



Pull Request #6536 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 483fd206779feb80744ca1cfc09886bb05f0c650 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-16] Comment by **xamarinc**



Fixed in version 8.0.0.560 (release-8.0)



Pull Request #6533 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 479f1c73b696df54a75eaae2eabde35a55f1753c (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-18] Comment by **xamarinc**



Fixed in version 8.0.0.1738 (release-8.0)



Author: Vsevolod Kukol  
 Commit: 502ba9946f7468a0f0b8b99911755aa7700cf0d0 (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-24] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #6533 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 479f1c73b696df54a75eaae2eabde35a55f1753c (xamarin/monodevelop)
